### ISSUE TITLE
Issue a new release for opencensus-ext-sqlalchemy

### ISSUE THREAD

**Issue by iensu** [census-instrumentation/opencensus-python#1175]
The current version of opencensus-ext-sqlalchemy still has a upper version limit. The latest release on pypi.org is 0.1.3, which was released over a year ago, so the fix is in this PR is not available yet: https://github.com/census-instrumentation/opencensus-python/pull/1142

```
#0 13.38 The conflict is caused by:
#0 13.38     xxxxx 0.1 depends on SQLAlchemy==1.4.42
#0 13.38     opencensus-ext-sqlalchemy 0.1.3 depends on SQLAlchemy<1.3.24 and >=1.1.14
```

**Comment by JeremyVriens** [comment]
Agreed, @lzchen when can we expect a release of the `opencensus-ext-sqlalchemy`? [A fix](https://github.com/census-instrumentation/opencensus-python/pull/1142) has already been merged November 17th last year.

**Comment by macieyng** [comment]
@lzchen @aabmass @hectorhdzg @jeremydvoss please take a look at this issue. I think that this urgent.

**Comment by bujesse** [comment]
This is a hard block on upgrading sqlalchemy for those of us using pip-compile and related tools.

**Comment by misupantea-uipath** [comment]
@aabmass, @hectorhdzg , @lzchen any chance we could do a release?

**Comment by SmirnGregHM** [comment]
Please make a new release!

For those struggling to install it from github directly, try
```
pip install git+https://github.com/census-instrumentation/opencensus-python/@e4d08047e38453057ef406c423bd2f6d73b6d6be#subdirectory=contrib/opencensus-ext-sqlalchemy
```

**Comment by lzchen** [comment]
https://opentelemetry.io/blog/2023/sunsetting-opencensus/

Opencensus Python is not supported anymore and will be archived next month. No new releases will be made for this project.

**Comment by RobertoPrevato** [comment]
Pity, @lzchen! I think the OpenCensus ecosystem for Python was better than the current OpenTelemetry ecosystem. The performance of OpenTelemetry libraries for Python is very disappointing, in my opinion. The code API in OpenCensus was intelligible and clear, the new one not so much.
